d3a_api_client/utils.py

```diff
@@ -344,3 +341,26 @@ def get_name_from_area_name_uuid_mapping(area_name_uuid_mapping, asset_uuid):
             if area_uuid == asset_uuid:
                 return area_name
 
+
+def get_simulation_id_from_config_uuid(domain_name: str, config_uuid: str,
+                                       auth_headers: dict) -> str:
+    """Get simulation_id from configuration by performing graphql query."""
+    query = '''
+        query get_sim_id{
+            readConfiguration(uuid:"''' + config_uuid + '''"){
+                simulationResults
+            }
+        }
+    '''
+    ret_val = execute_graphql_request(domain_name, query, headers=auth_headers)
+    if "errors" in ret_val:
+        raise Exception(f"Error when acquiring the simulation_id {ret_val['errors']}")
+    return ret_val["data"]["readConfiguration"]["simulationResults"]
+
+
+def get_auth_headers_for_requests(domain_name: str) -> dict:
```

Comment:
I suggest caching this return value per domain name, you can also add ttl
```suggestion
@lru_cache()
def get_auth_headers_for_requests(domain_name: str) -> dict:
```

Reply:
I dont know how often this method will be called. Probably only 10 times in a row on one day.
But I can add it.